acceptance/setup/package/pre-suite/020_install.rb

```diff
@@ -0,0 +1,8 @@
+step 'Install Bolt package' do
+  bolt_sha = ENV['SHA']
+  bolt_version = ENV['SUITE_VERSION']
```

Comment:
Looks like bolt_version isn't needed, can we drop the SUITE_VERSION variable? There's been a bunch of recent work to make that unnecessary.